src/pages/home/axios/updateTopics.ts

```
@@ -0,0 +1,110 @@
+};
+
+/**
+ * Create a new topic for a subject
+ * @param subjectId The ID of the subject to create a topic for
+ * @param title The title of the new topic
+ * @param classRecordId The ID of the class record to associate with this topic
+ * @returns The created topic or null if creation failed
+ */
+export const createTopic = async (subjectId: number, title: string, classRecordId?: number | null) => {
+  try {
+    const topicData: any = { 
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)